<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>503</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>443</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>443</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Solar Resource Estimation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Load Analysis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PV Array Sizing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Charge controller Sizing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inverter Sizing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solar Storage Sizing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recommended PV System</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="printPushButton">
       <property name="text">
        <string>&amp;Print</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
